lib/ssf/client.rb

```diff
@@ -27,69 +27,21 @@ def send_to_socket(message)
       @socket.send(message, 0)
     end
 
-    def new_root_span(operation: '', tags: {})
+    def new_span(operation: '', tags: {}, trace_id: nil, parent_id: nil)
```

Comment:
Also, we're going to want to do this by passing in the parent span, not the parent + trace id. Passing in the full span allows us to do things down the line like automatically propagate tags downward at the client level.


I'd also recommend putting the hash as the last parameter, so it goes `operation, span, tags`.